tools/mongo_client_in_k8s.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  labels:
+    run: mongoshclient
+  name: mongoshclient
+  namespace: armonik
+spec:
+  containers:
+  - name: mongosh
+    image: rtsp/mongosh
+    command: ["tail", "-f"]
+    args: ["/dev/null"]
+#    command: [mongosh]
```

Comment:
pourquoi pas le run de la command mongo ?

Reply:
La commande mongo doit être exécuté dans l'`exec` pour donner une session interactive à l'utilisateur.

Reply:
oui et je voulais faire ça  : kubectl run -it --remove mongoclient --image=rtsp/mongosh -- mongosh...
mais il faut montrer des drives pour les cred, du coup j'aurai voulu faire ça : 
kubectl run -f access-mongo-aws-as-admin.sh
mais on ne peut apparemment pas.
d'ou la méthode en 2 fois

Reply:
Il y a peut-être moyen avec l'option `--overrides` de `kubectl run`: https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#run

Reply:
ok I did it :))))